interface/main/calendar/modules/PostCalendar/pnuser.php

```diff
@@ -368,7 +368,7 @@ function postcalendar_user_search()
         // build our search query
         foreach ($keywords as $word) {
             if (!empty($sqlKeywords)) {
-                $sqlKeywords .= " $k_andor ";
+                $sqlKeywords .= " " . add_escape_custom($k) . "_andor ";
```

Comment:
@stephenwaite think you have a typo here,  **$k_andor** is a variable on line 126 so you'll need to escape the entire variable instead of **$k**.